doc/conf.py

```diff
@@ -12,6 +12,10 @@
 
 # ruff: isort: on
 
+requires_python = metadata("pygmt")["Requires-Python"]
+with pygmt.clib.Session() as lib:
+    requires_gmt = ">=" + lib.required_version
```

Comment:
Can make this an f-string
```suggestion
    requires_gmt = f">={lib.required_version}"
```